sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.aest.controller.assessApplicationOfferingChangeRequest.e2e-spec.ts

```diff
@@ -9,23 +9,31 @@ import {
 import {
   E2EDataSources,
   createE2EDataSources,
+  createFakeEducationProgramOffering,
+  createFakeStudentAppeal,
+  createFakeStudentAssessment,
+  createFakeUser,
   saveFakeApplication,
   saveFakeApplicationOfferingRequestChange,
 } from "@sims/test-utils";
 import {
   ApplicationOfferingChangeRequestStatus,
   ApplicationStatus,
   AssessmentTriggerType,
+  User,
 } from "@sims/sims-db";
+import { Repository } from "typeorm";
 
 describe("ApplicationOfferingChangeRequestAESTController(e2e)-assessApplicationOfferingChangeRequest", () => {
   let app: INestApplication;
   let db: E2EDataSources;
+  let userRepo: Repository<User>;
 
   beforeAll(async () => {
     const { nestApplication, dataSource } = await createTestingAppModule();
     app = nestApplication;
     db = createE2EDataSources(dataSource);
+    userRepo = dataSource.getRepository(User);
```

Comment:
The `db` should be used as a shortcut. We no longer need to use individual `getRepository` methods for E2E tests.